kim_cnn/__main__.py

```diff
@@ -143,3 +144,8 @@ def evaluate_dataset(split_name, dataset_cls, model, embedding, loader, batch_si
     else:
         raise ValueError('Unrecognized dataset')
 
+    if args.onnx:
+        dummy_input = next(iter(dev_iter)).text
```

Comment:
Could we somehow define the dummy_input shape through the argument parser ? Because we could not control the shape (batch length, sentence length) through `dev_iter`